src/ci_rdm/ci_rdms.cc

```diff
@@ -968,7 +978,7 @@ void CI_RDMS::compute_3rdm_op(std::vector<double>& tprdm_aaa, std::vector<double
                 const double& sign_stu = std::get<1>(detI) > 0.0 ? 1.0 : -1.0;
                 const size_t& I = std::get<0>(detI);
 
-                double el = evecs_->get(I, root1_) * evecs_->get(J, root2_) * sign_pqr * sign_stu;
+                double el = evecs_->get(J, root1_) * evecs_->get(I, root2_) * sign_pqr * sign_stu;
```

Comment:
Are there tests that check for these changes?

Reply:
No. The old implementation assumes Hermitian RDMs, which (I think) are not guaranteed for transition RDMs. I will see if I can come up with a test case for transition RDMs

Reply:
So I added the tests of DETCI dipoles in `dsrg-mrpt2-8-sa`, which should tell the difference.